draft-thoji-scone-trone-protocol.md

```diff
@@ -310,6 +310,104 @@ This transport parameter is valid for QUIC versions 1 {{QUIC}} and 2
 transport parameters, and frame types registries established in {{Sections 22.2,
 22.3, and 22.4 of QUIC}}.
 
+# TRONE Indication {#trone-indication}
+
+QUIC endpoints can signal potential support for TRONE before the completion of
+the QUIC handshake by attaching a TRONE indication packet after a QUIC Initial
+packet in the first UDP datagram. The TRONE indication provides an
+opportunistic signal to network elements that the client might support TRONE.
+Network elements can use this as an early hint, but must await confirmation of
+TRONE support by observing a full TRONE packet after the handshake completes.
+
+## TRONE Indication Packet
+
+A TRONE indication packet has a format with fully reversed field order compared
+to TRONE packets to facilitate detection by parsing from the end of the packet.
+
+{{fig-trone-indication-packet}} shows the format of the TRONE indication packet:
+
+~~~ artwork
+TRONE Indication Packet {
+  Source Connection ID (0..2040),
+  Source Connection ID Length (8),
+  Destination Connection ID (0..2040),
+  Destination Connection ID Length (8),
+  Version (32) = 0xTRONE1 or 0xTRONE2,
+  Rate Signal (6) = 0x3F,
```

Comment:
I think there is a question around what TRONE indication packets signal.

IIRC, TRONE is unidirectional.

Assuming the use of a TRONE indication packet signal that the sender is capable of _receiving_ TRONE packets, If so, I do not think need a rate signal here (and separately, I think we can use a different version number for indication packets).

Reply:
Agreed, I think all the fields here are likely excessive, but kept them in for now so we can discuss further. A new version is probably fine in terms of avoiding false positives.